inc/Engine/Cache/Config/ConfigSubscriber.php

```diff
@@ -47,6 +47,29 @@ function ( $uri ) {
 	 * @return void
 	 */
 	public function regenerate_config_file() {
+		$cache_reject_uri = $this->match_pattern_with_permalink_structure( get_rocket_option( 'cache_reject_uri', [] ) );
+		update_rocket_option( 'cache_reject_uri', $cache_reject_uri );
```

Comment:
We could use our Options and Options_Data classes instead of the helper functions